components/builder-depot/src/error.rs

```diff
@@ -54,8 +64,12 @@ impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let msg = match *self {
             Error::BadPort(ref e) => format!("{} is an invalid port. Valid range 1-65535.", e),
+            Error::CaughtPanic(ref msg, ref source) => {
```

Comment:
Seems ominous.. is this actually a thing?

Reply:
Nope I don't believe so, I think I must have added this in an early iteration and forgotten to remove it.